### PR TITLE
Add cities, Mar 26 2016

### PR DESCRIPTION
**Pull request by hallaathrad**

### CODE REVIEW

File: List-of-Free-Code-Camp-city-based-Campsites.md [hunk 1]
```diff
@@ -465,6 +465,7 @@ If your city isn't listed here, you should [[create a Campsite for your city|How
   - Tamil Nadu
     - [Chennai](https://www.facebook.com/groups/free.code.camp.chennai/)
     - [Coimbatore](https://www.facebook.com/groups/free.code.camp.coimbatore)
+    - [Madurai](https://www.facebook.com/groups/free.code.camp.maduraicity/)
```
Comment (abhisekp) [inline List-of-Free-Code-Camp-city-based-Campsites.md]: weird url name ~~maduraicity~~ madurai

File: List-of-Free-Code-Camp-city-based-Campsites.md [hunk 2]
```diff
@@ -1228,6 +1228,7 @@ If your city isn't listed here, you should [[create a Campsite for your city|How
     - [Asheville](https://www.facebook.com/groups/free.code.camp.asheville/)
     - [Charlotte](https://www.facebook.com/groups/free.code.camp.charlotte.nc/)
     - [Durham](https://www.facebook.com/groups/free.code.camp.durham/)
+    - [Elizabeth City](https://www.facebook.com/groups/free.code.camp.Elizabeth.City/)
```
Comment (abhisekp) [inline List-of-Free-Code-Camp-city-based-Campsites.md]: Weird name here too

Comment (abhisekp) [inline List-of-Free-Code-Camp-city-based-Campsites.md]: Weird name here too

Comment (hallaathrad) [inline List-of-Free-Code-Camp-city-based-Campsites.md]: https://en.wikipedia.org/wiki/Elizabeth_City,_North_Carolina

File: List-of-Free-Code-Camp-city-based-Campsites.md (hunk 1 above)
Comment (hallaathrad) [inline List-of-Free-Code-Camp-city-based-Campsites.md]: again... I don't create those URLs and FB doesn't let you change them once created.
Besides, seriously. those URLs don't matter. people won't type the whole thing manually. 
 :+1:

Comment (abhisekp) [inline List-of-Free-Code-Camp-city-based-Campsites.md]: @hallaathrad ah! I didn't know one can't change the url once created. I thought I would point out and then you can follow it up and ask the admin to modify the url.
It's alright then. I won't comment about the url. :+1:

**Pull request merged**